charmcraft/templates/init/README.md

```diff
@@ -0,0 +1,16 @@
+# {{ name }}
+
+## XXX: fill this out.
+
+## developing
+
+Create a virtualenv in your preferred way, and install `requirements-dev.txt`.
+For example,
+
+    pipenv install --dev
```

Comment:
I'll make five different readme each with a different way of creating and using a virtualenv :-p

Reply:
I've always used `python3 -m venv  .venv`
because that comes with python. I support pipenv means you don't have to create and then install it?

Reply:
Yes, use `python3 -m venv` (as it's always there) or `virtualenv` (which is the canonical tool to create venvs).

Reply:
Also:

https://chriswarrick.com/blog/2018/07/17/pipenv-promises-a-lot-delivers-very-little/
https://www.reddit.com/r/Python/comments/chkah3/is_pipenv_dead_why_has_the_project_stopped/

Reply:
On Debian-derived `venv` is not always there (you need to install `python3-venv`, which is exactly the same amount of work as installing `virtualenv` except the latter has `command-not-found` support). We might as well go with the 'canonical' one.